packages/sdk/ts/deploy/utils.ts

```diff
@@ -24,6 +24,6 @@ export const deployFactoryWithLinkedLibraries = async ({
 
   const contractFactory = await createContractFactory(abi, bytecode, signer);
 
-  const contractFactoryContract = await deployContractWithLinkedLibraries(contractFactory, ...(args ?? []));
+  const contractFactoryContract = await deployContractWithLinkedLibraries(contractFactory, signer, ...(args ?? []));
```

Comment:
The coordinator service is using  
```javascript
{deployMaci } from "@maci-protocol/sdk";
```

and `deployMaci` uses this `deployFactoryWithLinkedLibraries` that accepts a signer but was not sending it to the internal `deployContractWithLinkedLibraries`